README.md

```diff
@@ -1,24 +1,314 @@
 # CSS Fundamentals
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/-k-1TU8qq0Q?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe><p><a href="https://www.youtube.com/watch?v=-k-1TU8qq0Q">CSS Fundamentals</a></p>
+## Problem Statement
 
+You know the basics of CSS and how to use it to style HTML pages. But you want
+to customize your presentation further. Fortunately, CSS has several fundamental
+powers to explore, including selector types, color values and comments.
+
+## Objectives
+
+1. Identify CSS syntax
+2. Identify CSS use formats
+3. Utilize various types of CSS selectors
+4. Utilize various types of color values in CSS
+5. Recognize and implement CSS comments
+
+## Identify CSS Syntax
+
+Before we turn to the various ways we can extend CSS rules, let's go over the
+foundational syntax CSS uses.
+
+<img src="https://curriculum-content.s3.amazonaws.com/fewds/css-syntax.png" alt="CSS syntax diagram labeling selector, property and value" width="500" />
+
+We create a CSS rule by defining the selector, which matches the HTML element we
+want to style. Inside the curly braces we declare the property we want to change
+and, after the colon, we set the value we want to change that property to. We
```

Comment:
"Each property is written in the following form: property name, colon, and the value for that property."...